cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperConfiguration.java

```diff
@@ -19,17 +20,19 @@
 public class ZookeeperConfiguration extends AbstractConfiguration {
 
     private static final List<String> FORBIDDEN_OPTIONS;
-    private static final Map<String, String> DEFAULTS;
+    protected static final Map<String, String> DEFAULTS;
 
     static {
         FORBIDDEN_OPTIONS = asList(
                 ZookeeperClusterSpec.FORBIDDEN_PREFIXES.split(" *, *"));
 
-        DEFAULTS = new HashMap<>();
-        DEFAULTS.put("tickTime", "2000");
-        DEFAULTS.put("initLimit", "5");
-        DEFAULTS.put("syncLimit", "2");
-        DEFAULTS.put("autopurge.purgeInterval", "1");
+        Map<String, String> config = new HashMap<>();
+        config.put("tickTime", "2000");
+        config.put("initLimit", "5");
+        config.put("syncLimit", "2");
+        config.put("autopurge.purgeInterval", "1");
+        config.put("4lw.commands.whitelist", "*");
```

Comment:
Does this option exist even in old ZK versions? Or will it be at least ignored?

Reply:
I have tested it with the old ZK and it is ignored